junakowicz/bb-toto-bounce-back
# add mysql to ansible

fixes #4

This just adds sql with default username/password as root/root and no databases, so very bare bones and completely insecure/useless.

Testing:
- run `vagrant up` or `vagrant provision`
- run `vagrant ssh -c "mysql -u root -p root"`
- this should allow running sql queries, test e.g. `show databases;`
